com.unity.renderstreaming/Documentation~/en/components.md

```diff
@@ -19,9 +21,27 @@
 | *Credencial*                 | The credential to use when logging into the TURN server                                                                    |                                 |
 | *Credencial Type*            | This attribute specifies what kind of `credential` is to be used when connecting to the TURN server                        | Password                        |
 | **Interval**                 | Set the polling frequency (in seconds) to the signaling server <br>Valid only if you are using Http as the signaling protocol. | 5 (sec)                         |
-| **Streaming Size**           | The size of the frame buffer for the streaming                                                                             | 1280, 720                       |
-| **Capture Camera**           | Set the streaming camera                                                                                                   |                                 |
 | **Hardware Encoder Support** | Enable the checkbox to make this video encoding replace values that use a [hardware encoder](../index.md#hardware-encoder) | Enabled                         |
 | **Array Button Click Event** | Register browser button click events                                                                                       |                                 |
 | *Element Id*                 | Set an ID that is sent from browsers                                                                                       |                                 |
-| *Click*                      | Set button click event                                                                                                     |                                 |
\ No newline at end of file
+| *Click*                      | Set button click event                                                                                                     |                                 |
+
+## `CameraStreamer` Component
```

Comment:
This is supposed to be an English page, right ?

Reply:
Yes, I will request Chris translating it in English

Reply:
Ok